pylint/extensions/check_docstring.py

```diff
@@ -0,0 +1,65 @@
+from pylint.checkers.base import BaseChecker
+from pylint.interfaces import IAstroidChecker, HIGH
+from pylint.checkers.utils import check_messages
+
+
+class _BasicChecker(BaseChecker):
+    __implements__ = IAstroidChecker
+    name = 'basic'
+
+
+class DocStringAddicChecker(_BasicChecker):
+    """Checks format of docstrings based on PEP 0257
+    """
+    msgs = {
+        'C0198': ('Bad docstring quotes in %s, expected """, given %s',
+                  'bad-docstring-quotes',
+                  'Used when docstring do not have triple double quotes.'),
+        'C0199': ('First line empty in %s docstring',
+                  'docstring-first-line-empty',
+                  'Used when blank line from the beginning of the docstring.'),
+        }
+
+    @check_messages('docstring-first-line-empty', 'bad-docstring-quotes')
+    def visit_module(self, node):
+        self._check_docstring('module', node)
+
+    def visit_classdef(self, node):
+        self._check_docstring('class', node)
+
+    def visit_functiondef(self, node):
+        ftype = node.is_method() and 'method' or 'function'
+        self._check_docstring(ftype, node)
+
+    visit_asyncfunctiondef = visit_functiondef
+
+    def _check_docstring(self, node_type, node):
```

Comment:
You might get more value by using the token checker, but you'll have to check if it works for your case.